tests/e2e/android-kotlin/readme.md

```diff
@@ -0,0 +1,46 @@
+# RudderTyper - Android Kotlin
+
+This is a sample Android Kotlin app designed to demonstrate how to use the RudderStack Android SDK to track the RudderTyper generated events.
+
+## Prerequisites
+
+- Download, install and setup [Android Studio](https://developer.android.com/studio/install).
+
+## Getting Started
+
+### Step 1: Open the `android-kotlin` folder, located under `tests/e2e`, in Android Studio
+
+### Step 2: Configure RudderStack Properties
+- Rename `rudderstack.properties.sample` to `rudderstack.properties`.
+- Update the `writeKey`, `dataPlaneUrl`, and `controlPlaneUrl` with your specific values.
+
+### Step 3: Sync Gradle
+
+- Sync the Gradle files by clicking on the `Sync Now` link in the top right corner of the Android Studio window. Or, you can click on the `File` menu and select `Sync Project with Gradle Files`:
+
+    ![syncGradle](images/syncGradle.png)
+
+### Step 4: Add Generated Files
```

Comment:
Can we add some automated script that can build the RudderTyper and add the output to this sample app?

Reply:
Yes, use @akashrpo's help to get it working.
The sample apps should be ready to test with minimal effort.

Reply:
`tests/e2e/android-java/typewriter.yml` <- This is deleted, but we can have `ruddertyper.yml` here and we just need to run the existing scripts `npx ruddertyper prod or dev` to get the files in the desired location as per the path value in yml file.

Reply:
I've added the relevant steps to generate the corresponding ruddertyper files. Please go through the updated readme.
I've also added the output to the sample app.

PS: I've replaced all the credentials with dummy values.